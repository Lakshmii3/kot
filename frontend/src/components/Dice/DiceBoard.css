.dice_container {
  display:flex;
  
  flex-direction: column;
  font-family: 'Open Sans', sans-serif;
}

.selected {
  background: green;
  color: black;
}

.unselected {
  background: transparent;
  color: black;
}