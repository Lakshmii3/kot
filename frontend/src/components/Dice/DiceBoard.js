import React from 'react';

import { List, Typography } from 'antd';
import "./DiceBoard.css";

const DiceBoard = (props) => {
    return (
        <div>
            <List
                // header={<div>Header</div>}
                // footer={<div>Footer</div>}
                bordered
                dataSource={props.data}
                renderItem={item => (
                    <List.Item>
                        {item}
                    </List.Item>
                )}
            />
        </div>
    );
}


export default DiceBoard;