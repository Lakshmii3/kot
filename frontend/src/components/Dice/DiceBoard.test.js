import React from 'react';
import { shallow } from 'enzyme';

import DiceBoard from './DiceBoard';

describe('Verify DiceBoard layout template', () => {
    it('DiceBoard should render correctly', () => {
        const errorPage = shallow(<DiceBoard />);
        expect(errorPage).toMatchSnapshot();
    });
})