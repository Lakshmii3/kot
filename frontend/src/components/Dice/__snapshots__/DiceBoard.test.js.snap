// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify DiceBoard layout template DiceBoard should render correctly 1`] = `
<div>
  <List
    bordered={true}
    dataSource={Array []}
    loading={false}
    pagination={false}
    renderItem={[Function]}
    split={true}
  />
</div>
`;
